### ISSUE TITLE
Communities - CHAT - los banners de las communities salen en blanco

### ISSUE THREAD

**Issue by IsbetVR** [Fermat-ORG/beta-testing-program#481]
#### Communities 

-----

##### Reporte de error: 

   *El error sucede cuando entro al "Communities", los banners salen todos en blanco. 

------
#### Pantalla 1

![screenshot_10](https://cloud.githubusercontent.com/assets/19142276/15838862/47ff20c6-2c0e-11e6-8646-10843cad1f3d.jpg)

------

#### Dispositivo desde donde se realizo la prueba

* Marca: Genymotion
* Modelo: Google Nexus 4 (320 dpi)
* Version de Android: 5.0.0

**Comment by IsbetVR** [comment]
@darkestpriest @sanro14

**Comment by darkestpriest** [comment]
¿estabas ejecutando algo adicional? Como el music player o algo así.

**Comment by IsbetVR** [comment]
no, nada @darkestpriest sólo abrí la communities

**Comment by juanaso** [comment]
Son los únicos banners que no aparecen?

**Comment by juanaso** [comment]
Y siempre salen en blanco? O solo algunas veces

**Comment by IsbetVR** [comment]
todos los de la communities, ninguno sale.
Siempre, desde que instala la apk salen en blanco

**Comment by juanaso** [comment]
Osea que no sólo chat y artist, sino que crypto currencies y digital asset, estaban faltantes de banners?

**Comment by lostwood** [comment]
trato de reproducir tu error pero siguiendo los pasos que dices no me pasa lo que comentas, por lo menos con la segunda version no, revisar los pasos nuevamente

**Comment by IsbetVR** [comment]
@juanaso exacto! todos los banners salen en blanco, lo que pasa es que le tomé capture a esa parte de la pantalla. Pero son todos

**Comment by IsbetVR** [comment]
bueno, yo estaba usando la versión "v1u1" y con esa me dio ese error @lostwood

**Comment by darkestpriest** [comment]
No hemos podido reproducir el error @IsbetVR si nos puedes indicar con más detalle, no solo el dispositivo, ya que todos probamos con dicho dispositivo y a ninguno nos ha sucedido.
Por favor, indícanos si tienes otra versión de fermat instalada, si estás haciendo una instalación limpia, todos los detalles, no importa lo pequeño que sea, para intentar reproducir el error por acá y así buscar una solución. Si puedes enviarnos las carécterísticas de tu equipo donde emulas fermat, versión del emulador y si te ha pasado con cualquier otra prueba.

**Comment by IsbetVR** [comment]
@darkestpriest antes de instalar cualquier apk yo borro primero la data y luego instalo la apk con la que voy a trabajar. Instalé la apk tky-art, creé las identidades de tky y de art, tengo entendido que se pueden crear en un mismo dispositivo de acuerdo a una explicación por videollamada la semana pasada (no recuerdo con quien fue, estaba @sanro14 también). hago las conexiones correspondientes del las identidades de "art" con las de "tky" a través del perfil de cada identidad de "art". Luego me voy a "communitites" y pasa lo que les reporté. En realidad es todo el procedimiento que hice, no tenía nada adicional corriendo en el emulador.

**Comment by darkestpriest** [comment]
Con respecto a la PC donde lo emulas ¿que cartecterísticas tiene? La versión de genymotion también es un buen dato. ¿tienes otro programa corriendo en tu equipo, como chrome, firefox, android studio? Cualquier detalle es importante.

**Comment by IsbetVR** [comment]
es una i3, ram 4GB. la versión del genymotion es la más actualizada. firefox estaba correndo si mal no recuerdo. Nada fuera de lo común en realidad. @darkestpriest

**Comment by darkestpriest** [comment]
¿Que sistema operativo usas?

**Comment by IsbetVR** [comment]
Windows, pero no creo que tenga que ver, porque he probado otras apk y no me había pasado esto

**Comment by darkestpriest** [comment]
Si tiene muchísimo que ver @IsbetVR windows es un sistema operativo que no es eficiente con los recursos del equipo, con las características de tu equipo, más el emulador y el tamaño actual de fermat puede estar afectando el desempeño del emulador y el thread que muestra los banners se esté interrumpiendo por alguna razón, procesador ocupado por los múltiples procesos de windows, el antivirus revisando algo, y un extenso etc. ¿Tienes algun teléfono físico en el cual puedas probar y verificar que suceda el mismo error?

**Comment by MaviDavsa** [comment]
Hola chicos! Ya tengo una chica que lo esta probando en el tlf y no les salio el error! Sin embargo yo estaba haciendo unas pruebas ayer en la tarde y me sale este error... 

![ihoh](https://cloud.githubusercontent.com/assets/17273005/15900244/9c938180-2d68-11e6-8207-92980b333f71.JPG)


![google nexus 5 0](https://cloud.githubusercontent.com/assets/17273005/15900252/a3146d9e-2d68-11e6-8031-dcbf7140b536.JPG)

**Comment by MaviDavsa** [comment]
Chicos acabo de probar y si ya me salio! Quedo todo genial! Perooo... las imagenes de los banner se ven pixeladas! Entonces es necesario cambiarla! :) 

![dfgdfg](https://cloud.githubusercontent.com/assets/17273005/15900553/b284be36-2d69-11e6-91b5-398d16cf7241.JPG)

![google nexus 5 0](https://cloud.githubusercontent.com/assets/17273005/15900592/d88cdb68-2d69-11e6-91f0-13c94605a7b5.JPG)

**Comment by MaviDavsa** [comment]
@lostwood @darkestpriest

**Comment by darkestpriest** [comment]
@juanaso, por favor, échale un ojo a lo de los banners.

**Comment by IsbetVR** [comment]
En la versión v1u2. de la apk ya me salen las imágenes de los banner en Communities, pero se ven pixeladas las de la sección "Artist" en comparación con las otras (Chat, Crypto...)

![screenshot_29](https://cloud.githubusercontent.com/assets/19142276/16020485/52f76baa-317c-11e6-980a-b72fe92dca7c.jpg)


Pruebas realizadas en:
- [x] Emulador
- Google Nexus 4
- Versión Android 5.0.0

**Comment by darkestpriest** [comment]
Genial que ya funcionen los banners. Con respecto a las imagenes pixeleadas, ya se encargaron nuevos banners a los diseñadores para que dichos banners se vean con mejor.

**Comment by IsbetVR** [comment]
En la versión v1u3. el fondo de la leyenda de "Artist Community" así como el ícono no es igual al de "Fan Community". El de fan se ve más claro, me refiero al color del fondo, se ve opaco.

![screenshot_6](https://cloud.githubusercontent.com/assets/19142276/16199090/4c5ff9e2-36d6-11e6-903b-44dc0832f727.jpg)

- [x] Emulador
- Google Nexus 4
- Versión de android: 5.1.0
- 320 dpi
- API 22 768x1280

**Comment by lostwood** [comment]
Ok le voy a pedir a una persona del grupo de los diseñadores para ver que su opión, @isahq23 este es el issue que te comentaba, para que por favor lo revises gracias

**Comment by lostwood** [comment]
Que tal @arnaldos este es uno de los issue que te comente

**Comment by arnaldos** [comment]
Hola @lostwood estaba revisando el caso para esto el recurso que te pasaron cuanto mide ??

**Comment by lostwood** [comment]
Que tal el  de la fan community mide 481 × 235 px y el del artist 481 x  234 px

**Comment by arnaldos** [comment]
ya mi team esta mejorando esto para pasártelos y plicarlos

**Comment by isahq23** [comment]
@lostwood te pasé los banners por slack y también están en el repo
